R/partial_dependency_individual.R

```
@@ -130,8 +131,9 @@ calculate_pd_vimp_normed <- function(pd, ensemble_colname = "ensemble", epsilon
   cutpoint_sd <- pd[model != ensemble_colname,
                     list(model_sd = sd(prediction)),
                     by = "feature_val"]
-  min_sd <- min(cutpoint_sd$model_sd) + epsilon
-  return(pd_vimp/min_sd)
+  summary_sd <- summary_function(cutpoint_sd$model_sd) + epsilon
+  stopifnot(length(summary_sd)==1)
```

Might be better to return a helpful error message when `length(summary_sd) == 1`